bin/stress-test/src/main.rs

```diff
@@ -93,11 +123,12 @@ async fn main() {
 // ================================================================================================
 
 /// Seeds the store with a given number of accounts.
-async fn seed_store(data_directory: PathBuf, num_accounts: usize) {
+async fn seed_store(data_directory: PathBuf, num_accounts: usize, public_accounts_percentage: u8) {
```

Comment:
Let's move `seed_store()` and associated functions into a separate module - e.g., something like `src/seeding.rs`.